src/microbe_stage/TweakedProcess.cs

```diff
@@ -4,13 +4,7 @@
 /// <summary>
 ///   A concrete process that organelle does. Applies a modifier to the process
 /// </summary>
-/// <remarks>
-///   <para>
-///     This is a struct as this just packs a few floats and a single object reference in here. This allows much tighter
-///     data packing when this is used in lists.
-///   </para>
-/// </remarks>
-public struct TweakedProcess : IEquatable<TweakedProcess>
+public class TweakedProcess : IEquatable<TweakedProcess>
```

Comment:
This is a pretty major change to no longer have this as a struct, and will now have temporary memory allocation consequences.

Reply:
Apparently, because of how process updating works, this change was unnecessary

Reply:
That's actually very good to hear, so this PR might be able to be polished up easily, then.